cost/turbonomics/scale_virtual_machines_recommendations/aws/turbonomics_scale_virtual_machines.pt

```diff
@@ -0,0 +1,396 @@
+name "Turbonomics Scale Virtual Machine Recommendations AWS"
+rs_pt_ver 20180301
+type "policy"
+short_description "POC turbonomics policy for scaling virtual machines [README](https://github.com/flexera-public/policy_templates/tree/master/cost/turbonomics/scale_virtual_machines_recommendations) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+severity "low"
+category "Cost"
+default_frequency "daily"
+info(
+  version: "1.0",
+  provider: "AWS",
+  source: "Turbonomics",
+  service: "Compute",
+  policy_set: "Scale VMs",
```

Comment:
From [ticket](https://flexera.atlassian.net/browse/HCLIND-52):

Ensure that the generated incidents will be converted into spend recommendations, in the policy info block:

- set provider to a corresponding cloud provider

- set recommendation_type to usage reduction

- new field source set to Turbonomic

- policy_set set to “Rightsize Compute Instances”

Reply:
You are right. I'll keep this conversation open because something might be need to be change in my latest commit.